v1/nix/pkgs/fetchPip/fetchPip.nix

```diff
@@ -89,7 +93,7 @@
 
   # we use mitmproxy to filter the pypi responses
   pythonWithMitmproxy =
-    python.withPackages
+    python3.withPackages
```

Comment:
I believe we need to split those `withPackages` call onto both pythons.
With this change it uses the correct python python for the proxy script but the wrong one in l198: `      $pythonWithMitmproxy/bin/python $buildScript`

Just fixed that in my branch https://github.com/phaer/dream2nix/commit/a9b3d5bd53c5d9ecd4a53dec7acda6c0eb1904fb